multifidelity_end2end.py

```diff
@@ -83,7 +83,22 @@ def main():
         data_df["delta"] = data_df[args.hf_col_name] - data_df[args.lf_col_name]
         target_col_names = "delta"
 
-    
+    elif args.model_type == "transfer":
+
+        data_df = create_low_fidelity(data_df, args)
+        target_col_names = [args.lf_col_name, args.hf_col_name]
+        # Can seperate using the same function.
+        data_df, lf_train_smiles, hf_train_smiles = separate_hf_and_lf_train(data_df, train_smiles, args)
+        data_df, lf_val_smiles, hf_val_smiles = separate_hf_and_lf_train(data_df, val_smiles, args)
+        data_df, lf_test_smiles, hf_test_smiles = separate_hf_and_lf_train(data_df, test_smiles, args)
+
+        print("HF train size:", len(hf_train_smiles))
+        print("LF train size:", len(lf_train_smiles))
+        print("HF val size:", len(hf_val_smiles))
+        print("LF val size:", len(lf_val_smiles))
+        print("HF test size:", len(hf_test_smiles))
+        print("LF test size:", len(lf_test_smiles))
```

Comment:
can you also add the print statements of the relevant sizes for delta ML? I missed that in the last PR